src/components/CaptureViewer/CaptureVideoViewer.tsx

```diff
@@ -3,34 +3,58 @@ import classNames from 'classnames'
 import { withBlobPreviewUrl } from './hocs'
 import { trackException } from '../../Tracker'
 import style from './style.scss'
+import { WithTrackingProps } from '~types/hocs'
+import { useRef } from 'preact/hooks'
 
 type CaptureVideoViewerProps = {
   ariaLabel: string
   className?: string
   previewUrl?: string
   onVideoError: () => void
-}
+} & WithTrackingProps
 
 const CaptureVideoViewer: FunctionComponent<CaptureVideoViewerProps> = ({
   ariaLabel,
   className,
   previewUrl,
   onVideoError,
-}) => (
-  <div className={classNames(style.videoWrapper, className)}>
-    <video
-      aria-label={ariaLabel}
-      className={style.video}
-      src={previewUrl}
-      onError={(event) => {
-        const target = event.target as HTMLVideoElement
-        const error = target.error as MediaError
-        trackException(`${error.code} - ${error.message}`)
-        onVideoError()
-      }}
-      controls
-    />
-  </div>
-)
+  trackScreen,
+}) => {
+  const videoRef = useRef<HTMLVideoElement>(null)
+
+  const onPlay = () => trackScreen('video_play')
+  const onEnded = () => trackScreen('video_finished')
```

Comment:
Should we rename this to `video_playback_finished`?

Reply:
done